lib/infinispan.js

```diff
@@ -3,954 +3,952 @@
-       * A promise that will be completed with true if the mapping was removed,
-       * or false if the key did not exist.
-       * If the 'previous' option is enabled, it returns the value
-       * before removal or undefined if the key did not exist.
-       * @memberof Client#
-       * @since 0.3
-       */
-      remove: function(k, opts) {
-        var ctx = transport.context(SMALL);
-        logger.debugl(function() {return ['Invoke remove(msgId=%d,key=%s,opts=%s)',
-                                          ctx.id, u.str(k), JSON.stringify(opts)]; });
-        var decoder = p.decodePrevOrElse(opts, p.hasSuccess, p.complete(p.hasSuccess));
-        return futureKey(ctx, 0x0B, k, p.encodeKey(k), decoder, opts);
-      },
-      /**
-       * Conditional store operation that associates the key with the given
-       * value if the specified key is not already associated with a value.
-       *
-       * @param k {(String|Object)} Key with which the specified value is to be associated.
-       * @param v {(String|Object)} Value to be associated with the specified key.
-       * @param opts {?StoreOptions} Optional store options.
-       * @returns {module:promise.Promise.<(Boolean|String|Object)>}
-       * A promise that will be completed with true if the mapping was stored,
-       * or false if the key is already present.
-       * If the 'previous' option is enabled, it returns the existing value
-       * or undefined if the key does not exist.
-       * @memberof Client#
-       * @since 0.3
-       */
-      putIfAbsent: function(k, v, opts) {
-        var ctx = transport.context(MEDIUM);
-        logger.debugl(function() {return ['Invoke putIfAbsent(msgId=%d,key=%s,value=%s,opts=%s)',
-                                          ctx.id, u.str(k), u.str(v), JSON.stringify(opts)]; });
-        var decoder = p.decodePrevOrElse(opts, p.hasNotExecuted, p.complete(p.hasSuccess));
-        return futureKey(ctx, 0x05, k, p.encodeKeyValue(k, v), decoder, opts);
-      },
-      /**
-       * Conditional store operation that replaces the entry for a key only
-       * if currently mapped to a given value.
-       *
-       * @param k {(String|Object)} Key with which the specified value is associated.
-       * @param v {(String|Object)} Value expected to be associated with the specified key.
-       * @param opts {?StoreOptions} Optional store options.
-       * @returns {module:promise.Promise.<(Boolean|String|Object)>}
-       * A promise that will be completed with true if the mapping was replaced,
-       * or false if the key does not exist.
-       * If the 'previous' option is enabled, it returns the value that was
-       * replaced or undefined if the key did not exist.
-       * @memberof Client#
-       * @since 0.3
-       */
-      replace: function(k, v, opts) {
-        var ctx = transport.context(MEDIUM);
-        logger.debugl(function() { return ['Invoke replace(msgId=%d,key=%s,value=%s,opts=%s)',
+ 'use strict';
```

Comment:
@NeerajGartia21 please check the formatter, seems that every line has an extra space added at the beginning

Reply:
Oh, sorry while committing it was fine, don't know how it happened. I will correct it.

Reply:
> ...don't know how it happened

I had similar issue in the past. After uninstalling prettier, and add this to the vscode settings `"editor.tabSize": 2,` I never saw that kind of issue anymore.
(not sure if it is the same case)